tests/Type/data/model-relations.php

```diff
@@ -16,6 +16,7 @@
 function test(User $user, \App\Address $address, Account $account, ExtendsModelWithPropertyAnnotations $model, Tag $tag, User|Account $union)
 {
     assertType('App\Account', $user->accounts()->firstOrCreate([]));
+    assertType('App\Account', $user->accounts()->createOrFirst([]));
```

Comment:
Like you can see in the Github Actions this and other tests you added will fail in 9.x

So, you'd need to create a new file, for example `model-relations-l10` and then conditionally include it in `GeneralTypeTest`